openmrs-19/src/main/java/org/motechproject/openmrs19/config/Configs.java

```diff
@@ -37,7 +37,10 @@ public void add(Config config) {
         if (getByName(config.getName()) != null) {
             throw new ConfigurationAlreadyExistsException(config.getName());
         }
-
+        String openMrsUrl = config.getOpenMrsUrl();
```

Comment:
I wonder if this is the best place to handle that. What if I set URL with '/' via file configuration? Can we instead handle it at the retrieval of the URL from the settings facade?